user/models.py

```diff
@@ -0,0 +1,48 @@
+from django.db               import models
+
+from .validators import (
+    phone_number_validation_function,
+    password_validation_function
+)
+
+class User(models.Model): 
+    name = models.CharField(
+        max_length = 16
+        )
+    birth = models.DateField(
+        max_length = 16
+        )
+    identifier = models.CharField(
+        max_length = 50,
+        unique     = True
+        )
+    password = models.CharField(
+        max_length = 50,
+        validators = [password_validation_function]
+        )
+    gender = models.CharField(
+        max_length = 2
+        )
+    phone_number = models.CharField(
+        validators = [phone_number_validation_function],
+        max_length = 17,
+        unique     = True
+        )
+    email = models.EmailField(
+        max_length = 254,
+        unique     = True
+        )
```

Comment:
속성의 property 까지 개별 라인으로 구분하지는 않으셔도 될 것 같습니다.
열 분리의 목적은 중요한 대상을 가독성있게 보기 위함이기 때문인데, models.py 한 클래스에서 가장 중요한 것은 각 테이블이 어떤 속성을 갖는지 한 눈에 들어오도록 하는 것입니다. 
```suggestion
    name = models.CharField(max_length = 16)
    birth = models.DateField(max_length = 16)
    identifier = models.CharField(max_length = 50, unique = True)
...(중략) ...
```

예를 들어 `Class`를 `import` 할 때에는 개별 `Class`가 중요하고, 어떤 Class를 가져와서 사용하는지를 한 눈에 알아야 해서 모두 분리하도록 추천드리지만 클래스 내부에서는 합치셔도 좋습니다 :)